### PR TITLE
Bug 1550160 - Clean up unused card CSS

### PR DESCRIPTION
**Pull request by gvn**
Testing:

Compare `dev-test-all` between Nightly and this version to ensure layout hasn't regressed.

### CODE REVIEW

**Comment by punamdahiya** [comment]
For cards seen with hero component, removing flex from .meta is causing 16px upward shift of the info content 

With patch
patch-cleanup

On master
master

File: content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss [hunk 1]
```diff
@@ -83,9 +80,6 @@ $excerpt-line-height: 20;
   }
 
   .meta {
-    display: flex;
```
Comment (punamdahiya) [inline content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss]: flex-grow inside .info-wrap needs this display:flex

File: content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss [hunk 2]
```diff
@@ -83,10 +80,8 @@ $excerpt-line-height: 20;
...
-    flex-direction: column;
```
Comment (punamdahiya) [inline content-src/components/DiscoveryStreamComponents/DSCard/_DSCard.scss]: we need flex-direction to align context 'sponsored by..'. Only CSS unused and safe to remove is flex-grow:1

**Comment by gvn** [comment]
Should be good now...thanks!

**Pull request merged**